http/src/main/java/io/micronaut/http/ssl/ClientSslConfiguration.java

```diff
@@ -100,6 +102,25 @@ void setTrustStore(@Nullable DefaultTrustStoreConfiguration trustStore) {
         }
     }
 
+    /**
+     * @return Whether the client should disable checking of the remote SSL certificate. <b>Note: This makes the SSL
+     * connection insecure, and should only be used for testing. If you are using a self-signed certificate, set up a
+     * trust store instead.</b>
+     */
+    public boolean isInsecureTrustAllCertificates() {
+        return insecureTrustAllCertificates;
+    }
+
+    /**
+     * @param insecureTrustAllCertificates Whether the client should disable checking of the remote SSL certificate.
```

Comment:
Add some documentation that says this only applies if no trust store is configured